mkt/reviewers/tests/test_views.py

```diff
@@ -1300,21 +1301,39 @@ def test_public_sign_failure(self, sign_mock, json_mock,
 
 
 class TestReviewMixin(object):
+    # E.g commreply+12e0caffc4ca4174a6f62300c0ff180a@marketplace.firefox.com .
+    COMM_REPLY_RE = r'^commreply\+[a-f0-9]+\@marketplace\.firefox\.com$'
```

Comment:
I can, but I read in the docs https://docs.python.org/2/library/re.html#module-contents that the regex's are cached by match() automatically.